src/components/middle/immedia/Immedia.scss

```diff
@@ -0,0 +1,68 @@
+.MainImmedia {
+  width: 100%;
+}
+
+.ImmediaHeader {
+  min-height: 110px;
+  max-height: 180px;
+  background: transparent;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  width: 100%;
+}
+
+.ImmediaBackground {
+  display: none;
+}
+
+.Participants {
```

Comment:
Could participants have a `justify-content` property? Currently, I see a gap on the right side of the container. It would be good if they are ordered nicely. Either `center` or `space-evenly` would work.

Reply:
Done in 1abdb39